sqlite3_web/lib/src/database.dart

```diff
@@ -50,6 +50,12 @@ abstract class Database {
   /// stream is active.
   Stream<SqliteUpdate> get updates;
 
+  /// A relayed stream of events triggered by rollbacks from the remote worker.
+  ///
+  /// Updates are only sent across worker channels while a subscription to this
+  /// stream is active.
+  Stream<void> get rollbacks;
```

Comment:
It's not clear how we would best translate this feature to `sqlite3_web`. We can't provide commit hooks that would be able to revert a transaction because that would require an async roundtrip between the worker and the main page.

We could provide a "read-only" `Stream<void> get commits` stream that installs a `() => true` filter reporting events if necessary. But I'm also fine with not exposing this in `sqlite3_web` until it is needed (`sqlite3_web` also provides the option of compiling custom workers where users have full control over the database, so they could install commit / rollback filters on the database if they need to).